src/components/dialogs/CancelPaymentErrorDialog.vue

```diff
@@ -36,13 +36,14 @@
 <script lang="ts">
 import Vue from 'vue'
 import { Component, Prop, Emit } from 'vue-property-decorator'
-import { mapGetters } from 'vuex'
+import { mapState } from 'pinia'
 import { ContactInfo } from '@/components/common'
+import { useRootStore } from '@/stores/rootStore'
 
 @Component({
   computed: {
     // Property definition for runtime environment.
-    ...mapGetters(['isRoleStaff'])
+    ...mapState(useRootStore, ['isRoleStaff'])
```

Comment:
Oh no, there's more of these. Sigh.

Reply:
History lesson: This is left over from early days where most of the components in this repo used "options" API to declare themselves. What we have here is partially options API and partially class API (as per line 50).

Part 2: New components in Vue3 seem to prefer options API (or composition API), but I believe there's lots of emerging libraries to keep using class API (which I prefer due their cleaner layout IMHO), so we don't need to change anything over at the moment.

Reply:
That makes sense. I had a feeling this was the case since this mainly appears in Filings UI which is the oldest one between our 3 UIs. I'll be replacing them with `@Getter`.

Reply:
I see you changed all except the 2 components that were writte in Options API -- perfect, thanks!